nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -171,7 +171,7 @@ def build_v2_url(response, repo_details)
           base_url: base_url,
           repository_url: base_url,
           versions_url: File.join(
-            base_url,
+            base_url.delete_suffix("/"),
```

Comment:
Would it be better to do this where we build up the repository_details? or are there valid uses for the trailing slash?

Reply:
I wasn't sure if it would ever be necessary in `base_url`, but I'm explicitly removing it here because we're calling `.join()` and I was seeing instances of a double slash.